lib/elastic_apm/rails.rb

```diff
@@ -36,6 +36,7 @@ def start(config)
            'action_dispatch'
          )
         require 'elastic_apm/spies/action_dispatch'
+        require 'elastic_apm/spies/action_controller'
```

Comment:
Do we want this (these even?) loaded like the other spies? So a user could easily `config.enabled_spies -= ['action_controller']`

Reply:
I dont know, what do you think? Would people want to disable these?

Reply:
As `caller` adds overhead, especially on JRuby, I'd say we ought to let people disable it.